usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -48,14 +48,14 @@
 
 
 TABLE_SPEC = {
-    "award_search": {
-        "model": None,
-        "source_table": None,
+    "award_search_testing": {
+        "model": AwardSearchView,
+        "source_table": "award_search",
         "destination_database": "rpt",
-        "partition_column": None,
-        "partition_column_type": None,
-        "delta_table_create_sql": award_search_sql_string,
-        "custom_schema": None,
+        "partition_column": "award_id",
+        "partition_column_type": "numeric",
+        "delta_table_create_sql": award_search_create_sql_string,
+        "custom_schema": "recipient_hash STRING, federal_accounts STRING",
```

Comment:
I think it may make sense to break the alphabetical ordering convention and move this under the testing comments at the bottom. Entries after the testing comment should remain in alphabetical order